src/components/Text/Text.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./text.css";
+
+export interface TextProps {
+  /**
+   * Text variants
+   */
+  variant?:
+    | "header"
+    | "subheader"
+    | "body"
+    | "body-small"
+    | "subtext"
+    | "header-alt"
+    | "subheader-alt"
+    | "body-alt"
+    | "subtext-alt";
```

Comment:
For the text component kyknya bs focus buat body aj. So the variant field is not needed anymore